rom/dev/src/flow/cold_reset/crypto.rs

```diff
@@ -36,21 +37,16 @@ impl Ecdsa384SignatureAdapter for Ecc384Signature {
 }
 
 /// DICE  Layer Key Pair
-#[derive(Debug)]
+#[derive(Debug, Zeroize)]
```

Comment:
Should these be `ZeroizeOnDrop`?

Reply:
We are explicitly calling the zeroize trait method on the critical structs, so just having Zeroize is good.

Reply:
It's worth considering in a future PR; I suspect we end up forgetting to zeroize many of these.

Reply:
Sounds good.